praw/models/reddit/subreddit.py

```diff
@@ -331,6 +331,7 @@ def submissions(self, start=None, end=None, extra_query=None):
 
     def submit(self, title, selftext=None, url=None, resubmit=True,
                send_replies=True):
+        # pylint: disable=line-too-long
```

Comment:
Sorry, I was overlooking line-too-long is solved 2 days ago (commit ec9b648a3668a06428f65646d7fe61408268b191).